mkdocs.yml

```diff
@@ -131,24 +130,27 @@ nav:
     - Rotate Secrets: "rotate-secrets.md"
     - Terraform: "terraform.md"
     - Authentication: "authentication.md"
-  - Development:
-    - Building: "development/building.md"
-    - Adding a feature: "development/adding_a_feature.md"
-    - Testing: "development/testing.md"
-    - New Kubernetes Version: "development/new_kubernetes_version.md"
-    - Developing using Docker: "development/Docker.md"
-    - Documentation Guidelines: "development/documentation.md"
-    - Hack Directory: "development/hack.md"
-    - How to update kOps API: "development/api_updates.md"
-    - Low level description on how kOps works: "development/how_it_works.md"
-    - Notes on Gossip design: "development/gossip.md"
-    - Notes on master instance sizing: "development/instancesizes.md"
-    - Our release process: "development/release.md"
-    - Releasing with Homebrew: "development/homebrew.md"
-    - Bazel: "development/bazel.md"
-    - Vendoring: "development/vendoring.md"
-    - Ports: "development/ports.md"
-    - Cluster Addons & Manager : "development/addons.md"
+  - Contributing:
+    - Getting Involved and Contributing: "contributing/contributing.md"
```

Comment:
This file is a bit oddly named, with the URL being `https://deploy-preview-10455--kubernetes-kops.netlify.app/contributing/contributing/` If we rename it to `contributing/index.md` does that shorten the URL to just `https://deploy-preview-10455--kubernetes-kops.netlify.app/contributing`?

Reply:
Yep, that worked! @rifelpet 
<img width="1350" alt="image" src="https://user-images.githubusercontent.com/12524503/103002362-bee41c80-44fc-11eb-9abe-485de85d4239.png">

Reply:
Pushed up! 😄 
https://deploy-preview-10455--kubernetes-kops.netlify.app/